build/release.js

```diff
@@ -47,12 +54,32 @@ module.exports = function( Release ) {
 		 * @param {Function} callback
 		 */
 		generateArtifacts: function( callback ) {
-			Release.exec( "npx grunt", "Grunt command failed" );
-			Release.exec(
-				"npx grunt custom:slim --filename=jquery.slim.js && " +
-					"npx grunt remove_map_comment --filename=jquery.slim.js",
-				"Grunt custom failed"
-			);
+			Release.exec( "npx grunt" );
+
+			for ( const slim of [ false, true ] ) {
+				for ( const esm of [ false, true ] ) {
+					if ( !slim && !esm ) {
+
+						// Special case already covered by the above `npx grunt`;
+						// skip it
+						continue;
+					}
+
+					const filename = `jquery${
+						slim ? ".slim" : ""
+					}.${ esm ? "m" : "" }js`;
```

Comment:
I don’t think you’re generating any `.mjs` files?

Reply:
Yeah, this whole block is actually not needed anymore, I'll drop it.